test/unit/Breadcrumbs.test.js

```diff
@@ -74,17 +73,17 @@ describe('Breadcrumbs', () => {
       });
 
       it('should contain two Link elements', () => {
-        expect(component.find('Link')).to.have.length(2);
+        expect(component.find('Link')).to.have.length(1);
       });
 
-      it('should link back to the regular search results page', () => {
+      xit('should link back to the regular search results page', () => {
```

Comment:
Left an extra `x` here.

Reply:
Which means this test breaks and I think this is one where you do want to check for the search results link item.

Reply:
This one is testing without search keyword. The describe says 'No search keyword'. So this tests the opposite. That's why I put the `x` there.